src/documents/ImageReordererAPI.md

```diff
@@ -0,0 +1,488 @@
+            <td>boolean</td>
+            <td>false</td>
+        </tr>
+
+        <tr>
+            <td>mergePolicy</td>
+            <td>an object describing how user options should be merged in with defaults
+For information on options merging, see [Options Merging](OptionsMerging.md)</td>
+            <td></td>
+            <td>
+                <pre>
+                    <code>
+mergePolicy: {
+    keysets: "replace",
+    "selectors.selectables":
+        "selectors.movables",
+    "selectors.dropTargets":
+        "selectors.movables"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Image Reorderer Options
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+            <th>Values</th>
+            <th>Default</th>
+        </tr>
+    </thead>
+
+    <tbody>
+        <tr>
+            <td><strong>Deprecated as of 1.1.2:</strong>
+afterMoveCallbackUrl</span></td>
+            <td>If an URL is provided with this option, the current state of the component model will be sent to that URL after a move is carried out using a default afterMove event handler. See [Deprecated - afterMoveCallback](http://wiki.fluidproject.org/display/Infusion13/Deprecated+-+afterMoveCallback) for info. </td>
+            <td>an URL</td>
+            <td>none</td>
+        </tr>
+    </tbody>
+</table>
+
+## Styling the Image Reorderer
+
+The Image Reorderer includes default CSS styles that it applies to the thumbnails. The application of styles is based on known class names. The _default_ class names are described below, and can be used by including the Image Reorderer stylesheet:
+
+```
+<link href="components/reorderer/css/imageReorderer.css" type="text/css" rel="stylesheet" media="all">;
+```
+
+NOTE that the default class names can be overridden with your own classes using the `styles` option: refer to the [Options](#options) section above.
+
+### Default Classes
+
+* `fl-reorderer-movable-default` - This class is applied to thumbnail elements in their default state.
+* `fl-reorderer-movable-selected` - This class is applied to the thumbnail that has been selected. The selected thumbnail item can be moved using keystrokes.
+* `fl-reorderer-movable-hover` - This class is applied to thumbnails when the mouse hovers over them.
+* `fl-reorderer-movable-dragging` - This class is applied to the thumbnail that is currently being moved.
+* `fl-reorderer-avatar` - This class is applied to the avatar, the image of the thumbnail as it is being dragged about by the mouse.
+* `fl-reorderer-dropMarker` - This class is applied to the drop target indicator when the mouse is used to drag a thumbnail.
+
+## Dependencies
+
+The Image Reorderer dependencies can be met by including the `MyInfusion.js` file in the header of the HTML file:
+```
+<script type="text/javascript" src="MyInfusion.js"></script>
+```
+
+Alternatively, the individual file requirements are:
```

Comment:
We should also add something describing the required CSS files – the ones that come with the component.